Packs/Infinipoint/Integrations/Infinipoint/Infinipoint.yml

```diff
@@ -841,7 +841,7 @@ script:
     - contextPath: Infinipoint.Scripts.execute.name
       description: Query name
       type: String
-  - name: infinipoint-get-non-compliance
+  - name: infinipoint-get-events
```

Comment:
How do we know this is the related command? And why not change it in the python file? this way people will not have to change playbooks and stuff.